modules/images-configuration-registry-mirror.adoc

```diff
@@ -66,53 +92,69 @@ In this example, you have a container image registry that is named `example.io`
-
-. Create the new `ImageContentSourcePolicy` object:
+<1> Indicates the API to use with this CR. This must be `config.openshift.io/v1`.
+<2> Indicates the kind of object according to the pull type, either:
+** `ImageDigestMirrorSet`. Pulls a digest reference image.
+** `ImageTagMirrorSet`. Pulls a tag reference image.
+<3> Indicates the type of image pull method, either:
+** `imageDigestMirrors` for an `ImageDigestMirrorSet` CR.
+** `imageTagMirrors` for an `ImageTagMirrorSet` CR. 
+<4> Indicates the name of the mirrored image registry and repository.
+<5> Optional. Indicates a secondary mirror repository for each target repository. If one mirror is down, the target repository can use another mirror.
+<6> Indicates the registry and repository source, which is the repository that is referred to in image pull specifications.
+<7> Optional. Indicates the fallback policy if the image pull fails:
+** `AllowContactingSource` allows continued attempts to pull the image from the source repository. This is the default. 
+** `NeverContactSource` prevents continued attempts to pull the image from the source repository.
+<8> Optional. Indicates a namespace inside a registry, which allows you to use any image in that namespace. If you use a registry domain as a source, the object is applied to all repositories from the registry.
+<9> Optional. Indicates a registry, which allows you to use any image in that namespace. If you specify a registry name, the object is applied to all repositories from a source registry to a mirror registry.
+<10> Pulls the image `registry.example.com/example/myimage@sha256:...` from the mirror `mirror.example.net/image@sha256:..`.
+<11> Pulls the image `registry.example.com/example/image@sha256:...` in the source registry namespace from the mirror `mirror.example.net/image@sha256:...`.
+<12> Pulls the image `registry.example.com/myimage@sha256` from the mirror registry `example.net/registry-example-com/myimage@sha256:...`. The `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry `mirror.example.net/registry-example-com`.
+
+. Create the new object:
 +
 [source,terminal]
 ----
 $ oc create -f registryrepomirror.yaml
 ----
 +
-After the `ImageContentSourcePolicy` object is created, the new settings are deployed to each node and the cluster starts using the mirrored repository for requests to the source repository.
+After the object is created, the Machine Config Operator (MCO) cordons the nodes as the new settings are deployed to each node. The MCO restarts the nodes for an `ImageTagMirrorSet` object only. The MCO does not restart the nodes for `ImageDigestMirrorSet` objects. When the nodes are uncordoned, the cluster starts using the mirrored repository for requests to the source repository.
 
 . To check that the mirrored configuration settings, are applied, do the following on one of the nodes.
```

Comment:
please remove the first comma: 
To check that the mirrored configuration settings are applied, do the following on one of the nodes.